client/src/test/java/io/pravega/client/control/impl/ControllerImplTest.java

```diff
@@ -2301,6 +2319,22 @@ public void testKVTablesInScope() {
                 e -> Exceptions.unwrap(e) instanceof RuntimeException);
     }
 
+    @Test
+    public void testGetKeyValueTableConfiguration() {
+        CompletableFuture<KeyValueTableConfiguration> kvtConfig = controllerClient.getKeyValueTableConfiguration("scope1", "kvtable1");
+        assertEquals(Integer.BYTES, kvtConfig.join().getPartitionCount());
```

Comment:
May I suggest setting different values for these fields? That way you can verify you aren't cross-assigning them by accident.

Reply:
Set different values as suggested.